Sources/Instructions/Extra/Default Views/CoachMarkBodyDefaultView.swift

```diff
@@ -82,12 +95,63 @@ private extension CoachMarkBodyDefaultView {
+                hintLabel.bottomAnchor.constraint(equalTo: bottomAnchor, constant: -spacing),
+                hintLabel.leftAnchor.constraint(equalTo: leftAnchor, constant: spacing),
+                hintLabel.rightAnchor.constraint(lessThanOrEqualTo: rightAnchor, constant: -spacing)
+            ])
+        case .topLeft:
+            labelStackView.addSubview(hintLabel)
+            labelStackView.addSubview(nextLabel)
+
+            NSLayoutConstraint.activate([
+                nextLabel.topAnchor.constraint(equalTo: topAnchor, constant: spacing),
+                nextLabel.bottomAnchor.constraint(equalTo: hintLabel.topAnchor, constant: -spacing),
+                nextLabel.leftAnchor.constraint(equalTo: leftAnchor, constant: spacing),
+                hintLabel.bottomAnchor.constraint(equalTo: bottomAnchor, constant: -spacing),
+                hintLabel.leftAnchor.constraint(equalTo: leftAnchor, constant: spacing),
+                hintLabel.rightAnchor.constraint(lessThanOrEqualTo: rightAnchor, constant: -spacing)
+            ])
+        case .bottomRight:
+            labelStackView.addSubview(hintLabel)
+            labelStackView.addSubview(nextLabel)
+
+            NSLayoutConstraint.activate([
+                hintLabel.topAnchor.constraint(equalTo: topAnchor, constant: spacing),
+                hintLabel.bottomAnchor.constraint(equalTo: nextLabel.topAnchor, constant: -spacing),
+                hintLabel.leftAnchor.constraint(equalTo: leftAnchor, constant: spacing),
+                hintLabel.rightAnchor.constraint(lessThanOrEqualTo: rightAnchor, constant: -spacing),
+                nextLabel.bottomAnchor.constraint(equalTo: bottomAnchor, constant: -spacing),
+                nextLabel.rightAnchor.constraint(equalTo: rightAnchor, constant: -spacing)
+            ])
+        case .bottomLeft:
+            labelStackView.addSubview(hintLabel)
+            labelStackView.addSubview(nextLabel)
+
+            NSLayoutConstraint.activate([
+                hintLabel.topAnchor.constraint(equalTo: topAnchor, constant: spacing),
+                hintLabel.bottomAnchor.constraint(equalTo: nextLabel.topAnchor, constant: -spacing),
+                hintLabel.leftAnchor.constraint(equalTo: leftAnchor, constant: spacing),
+                hintLabel.rightAnchor.constraint(lessThanOrEqualTo: rightAnchor, constant: -spacing),
+                nextLabel.bottomAnchor.constraint(equalTo: bottomAnchor, constant: -spacing),
+                nextLabel.leftAnchor.constraint(equalTo: leftAnchor, constant: spacing)
+            ])
+        case .none:
+            labelStackView.addArrangedSubview(hintLabel)
+            labelStackView.addArrangedSubview(separator)
+            labelStackView.addArrangedSubview(nextLabel)
+
+            separator.heightAnchor.constraint(equalTo: labelStackView.heightAnchor,
+                                              constant: -10).isActive = true
```

Comment:
Let's add the `.leading` case as well!

```suggestion
        case .trailing:
            labelStackView.addArrangedSubview(hintLabel)
            labelStackView.addArrangedSubview(separator)
            labelStackView.addArrangedSubview(nextLabel)

            separator.heightAnchor.constraint(equalTo: labelStackView.heightAnchor,
                                              constant: -10).isActive = true
        case .leading:
            labelStackView.addArrangedSubview(nextLabel)
            labelStackView.addArrangedSubview(separator)
            labelStackView.addArrangedSubview(hintLabel)

            separator.heightAnchor.constraint(equalTo: labelStackView.heightAnchor,
                                              constant: -10).isActive = true
```